utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -93,6 +93,12 @@ class HtmlParser private constructor(
       htmlContent = modifiedHtmlContent
     } else {
       htmlContentTextView.textDirection = View.TEXT_DIRECTION_LTR
+      val regex = Regex("""<oppia-noninteractive-image [^>]*>.*?</oppia-noninteractive-image>""")
+      val modifiedHtmlContent = rawString.replace(regex) {
+        val oppiaImageTag = it.value
+        """<div>$oppiaImageTag</div>"""
```

Comment:
This codes wraps the image in a div but it does not explicitly center it like in  line 91, so in RTL it might not be centered.

Reply:
You might want to add a comment in the code to explain the changes and make it clearer for other devs.

Reply:
> This codes wraps the image in a div but it does not explicitly center it like in line 91, so in RTL it might not be centered.

I have attached screenshot for RTL.